synthtool/py_templating_instructions/synth.py

```diff
@@ -0,0 +1,7 @@
+import synthtool as s
```

Comment:
Please add the usual Google copyright statement.

And add comments describing what you expect the reader to do with this file.